object_creator/fake_spell_window.h

```diff
@@ -1,6 +1,8 @@
 #ifndef CATA_OBJECT_CREATOR_FAKE_SPELL_WINDOW_H
 #define CATA_OBJECT_CREATOR_FAKE_SPELL_WINDOW_H
 
+#undef emit
```

Comment:
Isn't it handled by `QT_NO_KEYWORDS` define?

Reply:
Didn't know about it. Thanks!